Description: ignore upstream build/install steps
 They assume installing onto a live system, not for packaging.
 .
 cms (0.9+giteec7dca4-1) precise; urgency=low
 .
   * Initial release.
Author: Bernard Blackham <b-pkg@largestprime.net>

---

--- cms-0.9+giteec7dca4.orig/setup.py
+++ cms-0.9+giteec7dca4/setup.py
@@ -288,7 +288,3 @@ def install():
 
 if __name__ == "__main__":
     do_setup()
-    if "build" in sys.argv:
-        build()
-    if "install" in sys.argv:
-        install()
